server/api/league-table.get.ts

```diff
@@ -0,0 +1,8 @@
+const SPREADSHEET_ID = '1kUhXAJuCglttg6bUYQvcGilyXnUKAtW99Z8bJzQi0LU'
+const GOOGLE_API_KEY = 'AIzaSyBT2wft2I8F5Pp9qH_I22bTaeAU60IsKSI'
```

Comment:
not sure about exposing this Google API Key here. Is that your personal one? I think for now it's fine but we should potentially talk about where we transfer that too so that it's not dependent on a personal google api